requirements.txt

```diff
@@ -1,3 +1,12 @@
-datajoint>=0.13.0
-element-interface>=0.4.0
-plotly
+datajoint==0.14.0
```

Comment:
```suggestion
datajoint>=0.13.0
```

Reply:
Let's keep the previous version as the code can run with `datajoint>=0.13.0`.

Reply:
Sure sure Sir, Actually I made the change of replacing '==' to '>=' for higher versions in my cloned repo after our last meet.